### PR TITLE
sql-statement: update `flashback cluster` related msg

### PR DESCRIPTION
**Pull request by Defined2014**
### First-time contributors' checklist 

- [x] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [x] master (the latest development version)
- [x] v6.6 (TiDB 6.6 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.4 (TiDB 6.4 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: sql-statements/sql-statement-flashback-to-timestamp.md [hunk 1]
```diff
@@ -37,6 +37,7 @@ FlashbackToTimestampStmt ::=
 * `FLASHBACK CLUSTER` 指定的时间点不能存在未执行完成的 DDL 记录。若存在，TiDB 会拒绝该 DDL 操作。
 * 在执行 `FLASHBACK CLUSTER TO TIMESTAMP` 前，TiDB 会主动断开所有相关表上的连接，并禁止对这些表进行读写操作，直到 `FLASHBACK CLUSTER` 完成。
 * `FLASHBACK CLUSTER TO TIMESTAMP` 命令不能取消，一旦开始执行 TiDB 会一直重试，直到成功。
+* 在 `FLASHBACK CLUSTER` 执行期间，若有数据备份需求，只支持使用 br 并需要指定早于 `FLASHBACK CLUSTER` 开始时间的 `BackupTS` 进行备份，否则可能会报错或备份的数据存在一致性问题。
```
Comment (TomShawn) [inline sql-statements/sql-statement-flashback-to-timestamp.md:40]:
```suggestion
* 在 `FLASHBACK CLUSTER` 执行期间，若有数据备份需求，只支持使用 BR 并需要指定早于 `FLASHBACK CLUSTER` 开始时间的 `BackupTS` 进行备份，否则可能会报错或备份的数据存在不一致的问题。
```

File: sql-statements/sql-statement-flashback-to-timestamp.md [hunk 2]
```diff
@@ -37,6 +37,7 @@ FlashbackToTimestampStmt ::=
 * `FLASHBACK CLUSTER` 指定的时间点不能存在未执行完成的 DDL 记录。若存在，TiDB 会拒绝该 DDL 操作。
 * 在执行 `FLASHBACK CLUSTER TO TIMESTAMP` 前，TiDB 会主动断开所有相关表上的连接，并禁止对这些表进行读写操作，直到 `FLASHBACK CLUSTER` 完成。
 * `FLASHBACK CLUSTER TO TIMESTAMP` 命令不能取消，一旦开始执行 TiDB 会一直重试，直到成功。
+* 在 `FLASHBACK CLUSTER` 执行期间，若有数据备份需求，只支持使用 BR 并需要指定早于 `FLASHBACK CLUSTER` 开始时间的 `BackupTS` 进行备份，否则可能会报错或备份的数据存在不一致问题。
```
Comment (Defined2014) [inline sql-statements/sql-statement-flashback-to-timestamp.md:40]:
```suggestion
* 在 `FLASHBACK CLUSTER` 执行期间，若有数据备份需求，只支持使用 `BR BACKUP`，并需要指定早于 `FLASHBACK CLUSTER` 开始时间的 `BackupTS` 进行备份，使用其他方式可能会报错或备份的数据存在不一致问题。
```

File: sql-statements/sql-statement-flashback-to-timestamp.md [hunk 3]
```diff
@@ -37,6 +37,7 @@ FlashbackToTimestampStmt ::=
 * `FLASHBACK CLUSTER` 指定的时间点不能存在未执行完成的 DDL 记录。若存在，TiDB 会拒绝该 DDL 操作。
 * 在执行 `FLASHBACK CLUSTER TO TIMESTAMP` 前，TiDB 会主动断开所有相关表上的连接，并禁止对这些表进行读写操作，直到 `FLASHBACK CLUSTER` 完成。
 * `FLASHBACK CLUSTER TO TIMESTAMP` 命令不能取消，一旦开始执行 TiDB 会一直重试，直到成功。
+* 在 `FLASHBACK CLUSTER` 执行期间，若有数据备份需求，只支持使用 [BR 命令行工具](https://docs.pingcap.com/zh/tidb/dev/br-use-overview#%E9%80%9A%E8%BF%87%E5%91%BD%E4%BB%A4%E8%A1%8C%E5%B7%A5%E5%85%B7%E6%8E%A8%E8%8D%90)进行备份，并需要指定早于 `FLASHBACK CLUSTER` 开始时间的 `BackupTS` 进行备份。同时，在执行 FLASHBACK CLUSTER 时，[开启日志备份](https://docs.pingcap.com/zh/tidb/dev/br-pitr-guide#tidb-%E6%97%A5%E5%BF%97%E5%A4%87%E4%BB%BD%E4%B8%8E-pitr-%E4%BD%BF%E7%94%A8%E6%8C%87%E5%8D%97)的操作会失败，请等待 `FLASHBACK CLUSTER` 结束后再尝试开启日志备份
```
Comment (Defined2014) [inline sql-statements/sql-statement-flashback-to-timestamp.md:40]:
```suggestion
* 在 `FLASHBACK CLUSTER` 执行期间，若有数据备份需求，只支持使用 [BR 命令行工具进行快照备份](https://docs.pingcap.com/zh/tidb/dev/br-snapshot-guide)，并需要指定早于 `FLASHBACK CLUSTER` 开始时间的 `BackupTS` 进行备份。同时，在执行 `FLASHBACK CLUSTER` 时，[开启日志备份](https://docs.pingcap.com/zh/tidb/dev/br-pitr-guide)的操作会失败，请等待 `FLASHBACK CLUSTER` 结束后再尝试开启日志备份
```

File: sql-statements/sql-statement-flashback-to-timestamp.md [hunk 4]
```diff
@@ -37,6 +37,7 @@ FlashbackToTimestampStmt ::=
 * `FLASHBACK CLUSTER` 指定的时间点不能存在未执行完成的 DDL 记录。若存在，TiDB 会拒绝该 DDL 操作。
 * 在执行 `FLASHBACK CLUSTER TO TIMESTAMP` 前，TiDB 会主动断开所有相关表上的连接，并禁止对这些表进行读写操作，直到 `FLASHBACK CLUSTER` 完成。
 * `FLASHBACK CLUSTER TO TIMESTAMP` 命令不能取消，一旦开始执行 TiDB 会一直重试，直到成功。
+* 在 `FLASHBACK CLUSTER` 执行期间，若有数据备份需求，只支持使用 [BR 命令行工具进行快照备份](https://docs.pingcap.com/zh/tidb/dev/br-snapshot-guide)，并需要指定早于 `FLASHBACK CLUSTER` 开始时间的 `BackupTS` 进行备份。同时，在执行 `FLASHBACK CLUSTER` 时，[开启日志备份](https://docs.pingcap.com/zh/tidb/dev/br-pitr-guide)的操作会失败，请等待 `FLASHBACK CLUSTER` 结束后再尝试开启日志备份
```
Comment (Defined2014) [inline sql-statements/sql-statement-flashback-to-timestamp.md:40]:
```suggestion
* 在 `FLASHBACK CLUSTER` 执行期间，若有数据备份需求，只支持使用 [BR 命令行工具进行快照备份](https://docs.pingcap.com/zh/tidb/dev/br-snapshot-guide)，并需要指定早于 `FLASHBACK CLUSTER` 开始时间的 `BackupTS`。同时，在执行 `FLASHBACK CLUSTER` 时，[开启日志备份](https://docs.pingcap.com/zh/tidb/dev/br-pitr-guide)的操作会失败，请等待 `FLASHBACK CLUSTER` 结束后再尝试开启日志备份
```

**Review by bb7133 — COMMENTED** [review]
Need to change it to the internal reference link.

File: sql-statements/sql-statement-flashback-to-timestamp.md [hunk 5]
```diff
@@ -37,6 +37,7 @@ FlashbackToTimestampStmt ::=
 * `FLASHBACK CLUSTER` 指定的时间点不能存在未执行完成的 DDL 记录。若存在，TiDB 会拒绝该 DDL 操作。
 * 在执行 `FLASHBACK CLUSTER TO TIMESTAMP` 前，TiDB 会主动断开所有相关表上的连接，并禁止对这些表进行读写操作，直到 `FLASHBACK CLUSTER` 完成。
 * `FLASHBACK CLUSTER TO TIMESTAMP` 命令不能取消，一旦开始执行 TiDB 会一直重试，直到成功。
+* 在 `FLASHBACK CLUSTER` 执行期间，若有数据备份需求，只支持使用 [BR 命令行工具进行快照备份](https://docs.pingcap.com/zh/tidb/dev/br-snapshot-guide)，并需要指定早于 `FLASHBACK CLUSTER` 开始时间的 `BackupTS`。同时，在执行 `FLASHBACK CLUSTER` 期间，[开启日志备份](https://docs.pingcap.com/zh/tidb/dev/br-pitr-guide)的操作会失败，请等待 `FLASHBACK CLUSTER` 结束后再尝试开启日志备份。
```
Comment (Defined2014) [inline sql-statements/sql-statement-flashback-to-timestamp.md:40]:
```suggestion
* 在 `FLASHBACK CLUSTER` 执行期间，若有数据备份需求，只支持使用 [BR 命令行工具进行快照备份](/br/br-snapshot-guide.md)，并需要指定早于 `FLASHBACK CLUSTER` 开始时间的 `BackupTS`。同时，在执行 `FLASHBACK CLUSTER` 期间，[开启日志备份](/br/br-pitr-guide.md)的操作会失败，请等待 `FLASHBACK CLUSTER` 结束后再尝试开启日志备份。
```

**Review by bb7133 — APPROVED** [review]
LGTM

**Review by TomShawn — APPROVED** [review]
LGTM

**Comment by TomShawn** [comment]
translated in https://github.com/pingcap/docs/pull/12321.

**Pull request merged**